code/src/WijmoProvider/Features/Rows.ts

```diff
@@ -160,9 +160,15 @@ namespace WijmoProvider.Feature {
          * @param rowNumber Number of the row in which the class is going to be added.
          * @param className CSS class to add to the row.
          */
-        public addClass(rowNumber: number, className: string): void {
+        public addClass(
+            rowNumber: number,
+            className: string,
+            refresh = true
```

Comment:
A default value of true, is this what we really want to have?
I would prefer to have a default value false, so that we understand that we are on purpose triggering the invalidate of the grid...

Reply:
It is always better to go with the one that has more performance (without refreshing)... So I must agree

Reply:
done